domain/src/main/java/org/oppia/android/domain/translation/TranslationController.kt

```diff
@@ -340,6 +390,31 @@ class TranslationController @Inject constructor(
     asyncDataSubscriptionManager.notifyChange(APP_LANGUAGE_LOCALE_DATA_PROVIDER_ID)
   }
 
+  private fun retrieveLanguageContentCacheStore(
+    profileId: ProfileId
+  ): PersistentCacheStore<AppLanguageSelection> {
+    return cacheStoreMap.getOrPut(profileId) {
+      cacheStoreFactory.createPerProfile(
+        CACHE_NAME,
+        AppLanguageSelection.getDefaultInstance(),
+        profileId
+      ).also<PersistentCacheStore<AppLanguageSelection>> {
```

Comment:
You probably don't need to qualify the type parameter here, Kotlin should understand the type of ``it`` well enough.

Reply:
Thanks, updated.